userspace/libsinsp/container.cpp

```diff
@@ -84,8 +84,39 @@ const sinsp_container_info::container_mount_info *sinsp_container_info::mount_by
 
 sinsp_container_manager::sinsp_container_manager(sinsp* inspector) :
 	m_inspector(inspector),
-	m_last_flush_time_ns(0)
+	m_last_flush_time_ns(0),
+	m_unix_socket_path(string(scap_get_host_root()) + "/var/run/docker.sock"),
+	m_api_version("/v1.24")
```

Comment:
I like the change to embed the api version in the object, but if I recall I think the api version depends on the endpoint you're fetching, right? If that's the case, how about having multiple api versions here for the endpoints the object might need to fetch?

Reply:
I don't think we need an additional api version, `1.24` is fine for both the endpoints we might query (see https://docs.docker.com/engine/api/v1.24/#3-endpoints).